packages/odyssey-eslint/src/index.js

```diff
@@ -78,4 +78,5 @@ module.exports = {
       }
     },
   ],
+  extends: ["eslint:recommended", "plugin:prettier/recommended"],
```

Comment:
You'll notice above that I'm applying these `extends` on a per file extension basis. Reason is that typescript code will occasionally conflict with some of the `enlist:recommended` config and ships with its own baseline to use (see line 55 above). I believe we'll want to apply this a bit more carefully so that we avoid adding `enlist:recommended` for all files and add the prettier plugin where its necessary too